packages/kit/test/apps/basics/src/routes/load/mutated-server-data/+page.js

```diff
@@ -0,0 +1,7 @@
+import Component from './Component.svelte';
+
+export function load({ data }) {
+	// @ts-ignore we want to mutate the server load data
+	data.b = Component;
```

Comment:
It's not super important, but just so we have fewer moving pieces in this test, I would maybe make it try serializing `new class {}` rather than a dummy component.